src/main/webapp/app/quiz/manage/lifecyle-buttons/quiz-exercise-lifecycle-buttons.component.html

```
@@ -1,65 +1,69 @@
-            @if (batch.started && !batch.ended) {
-                <span> Running </span>
-            }
-            @if (batch.ended) {
-                <span> Done </span>
-            }
-            @if (!batch.started) {
-                <button id="instructor-quiz-start-{{ quizExercise().id }}-{{ batch.id }}" type="submit" (click)="startBatch(batch.id!)" class="btn btn-warning btn-sm me-1 mb-1">
-                    <fa-icon [icon]="faPlayCircle" />
-                    <span class="d-none d-md-inline" jhiTranslate="artemisApp.quizExercise.startQuiz"></span>
-                </button>
+<button id="instructor-quiz-start-{{ quizExercise().id }}" type="submit" (click)="startQuiz()" class="btn btn-success" [hidden]="!showStartButton()">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)